assets/js/templates/directives/admin_module_template.html

```diff
@@ -1,7 +1,7 @@
 <div class="container">
   <div class="row row-bottom-margin">
     <div class="col-xs-4">
-     <h2>Module View</h2>
+      <h2>Module View</h2>
```

Comment:
This space is actually good because it corrects the previously incorrect spacing